### DISCUSSION TITLE
Problem with Candy Machine Mint Request on latest Mobile Version (24.15.0)

### DISCUSSION THREAD

**Discussion by dvd900** [phantom/docs#287] [Help & Support]
### Summary

Hi All. I've had a NFT Mint running for a few months that just recently started displaying the error: Transaction simulation failed: Blockhash not found. It was working on both mobile and browser until about a week ago. This transaction still works on solflare, backpack, and glow.

For Context:
We rely on @metaplex-foundation/umi and @solana/wallet-adapter-base. Our Transaction looks like this:
```typescript
const nftMint = generateSigner(umi);
const transaction = await transactionBuilder()
  .add(
    mintV2(umi, {
      candyMachine: candyMachine.publicKey,
      candyGuard: candyGuard?.publicKey,
      nftMint,
      collectionMint: candyMachine.collectionMint,
      collectionUpdateAuthority: candyMachine.authority,
      mintArgs: {
        token2022Payment: some({ mint:tokenAddress, destinationAta}),
        mintLimit: some({id: 1})
      },
    })
  );
const { signature } = await transaction.sendAndConfirm(umi, {
  confirm: { commitment: "confirmed" },
});`
``` 
`       
`

The transaction seems to simulate fine as the cost, and the token issued looks correct in the approval window, we just error out almost immediately after the user approves the transaction. Any suggestions on what could of broken and how to make a compatible transaction for mobile?

### Steps to Reproduce

1. Create a candy-machine contract using sugar
2. Create a transaction (see above)
3. Notice behavior varies between mobile and browser

### Phantom Version

24.15.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
@dvd900 We haven't received any other reports specific to candy machine. We do have a fix coming out in the next version to address a block hash bug. Please let me know if the issue persists in 24.16.0
